contracts/token/SRC20.sol

```diff
@@ -5,45 +5,43 @@ import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 import "openzeppelin-solidity/contracts/cryptography/ECDSA.sol";
 import "./SRC20Detailed.sol";
 import "../interfaces/ISRC20.sol";
-import "../interfaces/ISRC20Owned.sol";
 import "../interfaces/ISRC20Managed.sol";
 import "../interfaces/ITransferRules.sol";
 import "../interfaces/IFeatured.sol";
 import "../interfaces/ISRC20Roles.sol";
 import "../interfaces/ISRC20.sol";
 import "../interfaces/ITransferRestrictions.sol";
+import "../interfaces/IAssetRegistry.sol";
 
 
 /**
  * @title SRC20 contract
  * @dev Base SRC20 contract.
  */
-contract SRC20 is ISRC20, ISRC20Owned, ISRC20Managed, SRC20Detailed, Ownable {
+contract SRC20 is ISRC20, ISRC20Managed, SRC20Detailed, Ownable {
     using SafeMath for uint256;
     using ECDSA for bytes32;
 
+    event RestrictionsAndRulesUpdated(address restrictions, address rules);
```

Comment:
Maybe move this event declaration to some interface.

Reply:
Moved to ISRC20.sol